docs/site/Model.md

```diff
@@ -921,6 +921,34 @@ class Product extends Entity {
 }
 ```
 
+### ENUM Property
```

Comment:
LGTM. Just wondering in the future will we keep this style or we'll only allow ` ENUM type` to be used? i.e is this 
a workaround for now? @raymondfeng

Reply:
`enum` is validated by TypeScript compiler as well as `AJV`. I think it's good as-is for now.